UKHomeOffice/cerberus-service
# COP-9617: Add tests to verify Risk Indicators on Rule matches section

## Description
Add tests to verify Risk Indicators on Rule matches section

GIVEN a task has matched rules
WHEN the task details are viewed
THEN the "Risk indicators" are displayed as the prototype for EACH matched rule

## To Test
npm run cypress:runner
run the test `Should check selector & rule matches details on task management page` from `task-management.spec.js`

`Should verify single task created for the same target with different versions when payloads sent without delay` from `task-version-details.spec.js`

## Developer Checklist

\* Required
